docs/CreateAzureADforAzureBot.md

```diff
@@ -2,19 +2,19 @@
 
 If you would like to authenticate against your own Azure AD for development or private use of the AzureBot code, you'll need to setup your own Azure AD application.  Here are the steps you need to follow:
 
-1. Create or Utilize an Azure Active Directory (AAD) Tenant
-2. Create a new Azure AD app on the AAD tenant
-3. Configure the AAD application for use with the AzureBot
-4. Configure App Multi-Tenancy (optional)
+- [Create or Utilize an Azure Active Directory (AAD) Tenant](#createtenant)
```

Comment:
I wasn't sure about numbering as they're not all needed, so thought about a list to be consistent. But I can change my dev setup one to add numbering, as long as both match, let me know what you prefer (numbering I suspect :))